src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -18,6 +21,177 @@
  */
 public final class Constants {
 
+    public static final class Arm {
+
+        //#region Subsystem Constants
+
+        // Array Indexes (Just to make things easier to read)
+        public static final int ARM = 0;
+        public static final int WRIST = 1;
+        public static final int CUBE = 0;
+        public static final int CONE = 1;
+        public static final int FRONT = 0;
+        public static final int BACK = 1;
+
+        // Feedforward
+        // Arm, Wrist
+        public static final double[] kS = {0.20642, .074798};
+        public static final double[] kG = {0.6697, 0.36214};
+        public static final double[] kV = {4.3735, 1.6743};
+        public static final double[] kA = {0.24914, 0.032177};
+        public static final double kG_WRIST = .36214; // (V)
+
+        // PID
+        // FIXME BOTH WRIST AND ARM NEED TO TEST PID (Wrist PID never tested)
+        // Arm, Wrist
+        public static double[] kP = {6.7868, 4.7391}; // 4.2736 for arm from sysid was tested and it worked fine (V / rad)
+        public static double[] kI = {0, 0}; // (V / (rad * s) )
+        public static double[] kD = {4.4327, 0.69517}; // 0 for arm from sysid was tested and it worked fine (V / (rad / s) )
```

Comment:
Why aren't we using the values from sysid? Were the new values arrived at via empirical tuning?